CONTRIBUTING.md

```diff
@@ -100,3 +100,41 @@ If a linting error is detected, the automated pull request checks will fail and
 ## Documentation
 
 Please check [docs/README.md](./docs/README.md) for instructions.
+
```

Comment:
Thanks for writing this! Could you please open a separate PR for this?

Reply:
Removed in commit https://github.com/mitmproxy/mitmproxy/commit/310d4b5b3b2d63fe64a922b80a7295740a329fc9

Reply:
@mhils I've opened a separate PR for the documentation: 

  * https://github.com/mitmproxy/mitmproxy/pull/4968